src/applications/managing-app-lifecycle.md

```diff
@@ -98,6 +98,22 @@ From deis-controller.local:peachy-waxworks
  * [new branch]      master     -> deis/master
 ```
 
+
+## Application Maintenance
+
+Maintenance mode for applications is useful to perform certain migrations or upgrades during which we don't want to serve client requests. Deis Workflow supports maintenance mode for app during which the access to the app is blocked. Blocking access to the application means all the requests to the app are served with error code of `503` and static maintenance page by the router but the app will still be running and one-off commands can still be run on them.
+
+To enable maintenance mode for app, use `deis maintenance`:
+
+    $ deis maintenance:on
+    Enabling maintenance for drafty-zaniness... done
+
+This will make all the request to the application serve 503 through the [Router][], but the application is still running. To disable maintenance mode:
```

Comment:
Do markdown links need to match in case as well? If so, `router` should be lower-case here. Also I might rephrase it a bit:
```
This will make the [router][] answer all requests to the application with a 503, although the app
is still running. To disable maintenance mode:
```